src/platform/Zephyr/BLEAdvertisingArbiter.cpp

```diff
@@ -88,7 +88,6 @@ CHIP_ERROR InsertRequest(Request & request)
 
     if (prev == nullptr)
     {
-        NotifyAdvertisingStopped(sys_slist_peek_head(&sRequests));
```

Comment:
This does not seem right.  The goal of this code is that if we are inserting the new request at the head of the list, that means we are stopping advertising on the current list head.  

It does need to account for the "no current list head" case, but removing the code altogether seems like a strange way to do said accounting.